module/Entra/config/ModuleSettings.json

```diff
@@ -4,6 +4,7 @@
     "newPrefix" : "Entra",
     "typePrefix" : "Microsoft.Open.",
     "destinationModuleName" : [
+      "Microsoft.Graph",
```

Comment:
Is the entire set of Microsoft.Graph submodules required? (I'm not yet clear about how this file is consumed and will keep reviewing.)